### PR TITLE
Disable Money Wash, Remove Vehicle List, Tweaked Jack Anim

### PR DESCRIPTION
**Pull request by Xrez31**
Removed 'raise' and 'lower', replaced with 'jack repair'.
Removed 'vehicle list' from Mechanic Actions
Transferred Disable Money Wash function from client\main.lua to server\main.lua

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -343,3 +343,8 @@ ESX.RegisterServerCallback('esx_mechanicjob:getPlayerInventory', function(source
 
 	cb({items = items})
 end)
+
```
Comment (alvinveroy) [inline server/main.lua]: Wrong implementation. Check Line 186 of client/main.lua

https://github.com/lockdownstudio/esx_mechanicjob/blob/1da4cceee7f6424a6f1ad7e74fbc27af1bc3e7af/client/main.lua#L186

File: client/main.lua [hunk 2]
```diff
@@ -1024,10 +1024,7 @@ AddEventHandler('esx:onPlayerSpawn', function(spawn)
 	isDead = false
 end)
 
---Disable Money Wash
```
Comment (alvinveroy) [inline client/main.lua:1038]: Calling Trigger to open Boss menu without correct conditions can be exploited.

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Wrong Implementation, Please change.

File: client/main.lua [hunk 3]
```diff
@@ -511,26 +511,15 @@ function OpenMobileMechanicActionsMenu()
...
 			local ped = PlayerPedId()
 			local coords = GetEntityCoords(ped)
 			local veh = ObjectInFront(ped, coords)
 			if DoesEntityExist(veh) then
 				if IsEntityAVehicle(veh) then
-					TriggerEvent('nk_repair:jackdown', ped, coords, veh)
+					TriggerEvent('nk_repair:JackRepair', ped, coords, veh)
 				else
-					ESX.ShowNotification('You must be near a ~y~Vehicle~s~!') -- TRANSLATE THIS - THAT SAY WHEN YOU DON'T HAVE ANY VEHICLE IN THE NEAR
+					ESX.ShowNotificatHAHAHAion('You must be near a ~y~Vehicle~s~!') -- TRANSLATE THIS - THAT SAY WHEN YOU DON'T HAVE ANY VEHICLE IN THE NEAR
```
Comment (alvinveroy) [inline client/main.lua]: Syntax Error.

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Syntax error. script will not execute.

File: client/main.lua (hunk 3 above)
Comment (Xrez31) [reply client/main.lua]: omg sorry po hahahaha. Will fix po.

File: server/main.lua (hunk 1 above)
Comment (Xrez31) [reply server/main.lua]: Updated implementation

**Comment by Xrez31** [comment]
"Fixed moneywash implementation, fixed Syntax Error on line 522"

File: client/main.lua [hunk 4]
```diff
@@ -185,7 +184,7 @@ function OpenMechanicActionsMenu()
 		elseif data.current.value == 'boss_actions' then
 			TriggerEvent('esx_society:openBossMenu', 'mechanic', function(data, menu)
 				menu.close()
-			end)
+			end), {wash = false}) -- set custom options, e.g disable washing
```
Comment (alvinveroy) [inline client/main.lua]: Unexpected closing of function. Should be:

```
end, {wash = false}) -- set custom options, e.g disable washing
```
See: https://github.com/ESX-Org/esx_society#usage

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Error on the script.

See: https://github.com/ESX-Org/esx_society#usage

for the right syntax.

**Comment by Xrez31** [comment]
Fixed wash disable syntax

**Pull request merged**